shared/dataflow/codeql/dataflow/TaintTracking.qll

```diff
@@ -56,7 +56,7 @@ module TaintFlowMake<
   private import MakeImpl<Location, DataFlowLang> as DataFlowInternal
   private import MakeImplStage1<Location, DataFlowLang> as DataFlowInternalStage1
 
-  private module AddTaintDefaults<DataFlowInternal::FullStateConfigSig Config> implements
+  private module MakeAddTaintDefaultsConfig<DataFlowInternal::FullStateConfigSig Config> implements
```

Comment:
Revert? I don't see a good reason for this change. Also, the double verb prefix becomes awkward. Whatever QL4QL rule that demands all modules implementing a data flow config to end in `Config` should instead be more lenient.

Reply:
Fair enough, I will revert it. However, I do think that the existing name is slightly confusing. The parameterized module is used to "make" a config module (maybe a better name is `MakeTaintDefaultsConfig`). In any case, I will just revert the change.